src/vm/eventpipeconfiguration.h

```diff
@@ -35,6 +35,12 @@ class EventPipeConfiguration
     // Perform initialization that cannot be performed in the constructor.
     void Initialize();
 
+    // Create a new provider.
+    EventPipeProvider* CreateProvider(const SString &providerName, EventPipeCallback pCallbackFunction, void *pCallbackData);
```

Comment:
Maybe change this signature to use default arguments of NULL for the callback functions to match the style of [eventpipeprovider.h:67](https://github.com/dotnet/coreclr/pull/14123/files#diff-2b9fac0903a4fae80811161f0c602ff1R67)

Reply:
This was actually intentional - the reason is that for the most part, no one should call this version of CreateProvider because they should always go through EventPipe::CreateProvider.